### DISCUSSION TITLE
Youtube videos exiting after a few minutes

### DISCUSSION THREAD

**Discussion by zapatistay** [FreshRSS/FreshRSS#5057] [Q&A] [answered]
Whenever I start watching a youtube video in an article, it exits after a few minutes of playing.

I guess this is due to freshrss refreshing itself every x minutes and the video play is killed when the refresh occurs. If so, Is there a setting where I can change that refresh period and make the video play longer? Better yet, is it possible to not have the video play killed ? If none of these work, I guess I'll have to open the video in a new page.

**Comment by Frenzie** [comment]
It's a little unclear to me what you mean by FreshRSS refreshing itself. Generally you'll just see a little notification at the top of the screen?

**Comment by zapatistay** [comment, marked as answer]
Sorry, my mistake. It was the Auto-refresh extension in action. It took me sometime to notice that.
